install/updates/73-certmap.update

```diff
@@ -0,0 +1,24 @@
+# Configuration for Certificate Identity Mapping
+dn: cn=certmap,cn=ipa,cn=etc,$SUFFIX
+default:objectclass: top
+default:objectclass: nsContainer
+default:objectclass: ipaConfigObject
```

Comment:
Now that the `ipaConfigString` is gone, `ipaConfigObject` is not necessary anymore.